File: C Program/Strong.c
#include<stdio.h>
 
int main()
 
{
 
 int num,i,fact,r,sum=0,temp;
 
 printf("Please enter a number to find strong number");
 
 scanf("%d",&num);
 
 temp=num;
 
 while(num)
 
 {
 
     i=1,fact=1;
 
     r=num%10;
 
     while(i<=r)
 
     {
 
        fact=fact*i;
 
        i++;
 
     }
 
     sum=sum+fact;
 
     num=num/10;
 
}
 
 
 
 if(sum==temp)
 
  printf("\nThe number %d is a strong number",temp);
 
 else
 
  printf("\nThe number %d is not a strong number",temp);
 
 return 0;
 
 }

File: python files/Strong.py
num = int(input(" Enter the Number:"))  
sum = 0  
temp = num  
  
while(temp > 0):  
    fact = 1  
    rem = temp % 10  
  
    for i in range(1, rem + 1):  
        fact = fact * i  
  
    print("Factorial of %d = %d" %(rem, fact))  
    sum = sum + fact  
    temp = temp // 10  
  
print("\n Sum of Factorials of a Given Number %d = %d" %(num, sum))  
      
if (sum == num):  
    print(" The given number is a Strong Number")  
else:  
    print(" The given number is not a Strong Number")  
